src/config_file.c

```diff
@@ -737,6 +737,8 @@ gboolean load_config(const gchar *filename, RaucConfig **config, GError **error)
 	} else if (ierror) {
 		g_propagate_error(error, ierror);
 		return FALSE;
+	} else if (ENABLE_STREAMING) {
+		g_message("Ignoring max-bundle-download-size, as streaming is enabled.");
```

Comment:
Since we do not really ignore it here, maybe saying
```suggestion
		g_message("Using max-bundle-download-size with streaming has no effect.");
```
is more consistent with the code? Any opinion? Otherwise, current version would be fine for me, too.